dpppt-backend-sdk/dpppt-backend-sdk-ws/src/test/java/org/dpppt/backend/sdk/ws/controller/GaenControllerTest.java

```diff
@@ -1354,6 +1388,64 @@ public void testMalciousTokenFails() throws Exception {
     assertTrue(authenticateError.contains("Unsigned Claims JWTs are not supported."));
   }
 
+  @Test
+  public void testUploadTodaysKeyWillBeReleasedTomorrow() throws Exception {
+    GaenRequest exposeeRequest = new GaenRequest();
+    List<GaenKey> keys = new ArrayList<>();
+    for (int i = 0; i < 30; i++) {
+      var tmpKey = new GaenKey();
+      tmpKey.setRollingStartNumber((int) UTCInstant.today().minusDays(i).get10MinutesSince1970());
```

Comment:
Again a very improbable race-condition when using `UTCInstant.today()`. You should put it at the beginning and then use `today` in the code.

Reply:
same as above